tools/ci_build/exclude_unused_ops.py

```diff
@@ -99,6 +104,9 @@ def extract_ops_from_graph(graph, operators, domain_opset_map):
                 model = onnx.load(model_path)
                 domain_opset_map = {}
 
+                if len(model.opset_import) == 0:
+                    continue
+
                 for opset in model.opset_import:
```

Comment:
Are both the additions to this file required? I would have expected the model to be invalid if there were no opset imports, or if there were nodes in the model with a domain not present in opset imports.

Reply:
Yes, when set model path to test_data script report errors which need be handled by the change.
My understanding is that some model there are stale.